bin/browser.js

```diff
@@ -1,6 +1,12 @@
 const puppeteer = require('puppeteer');
+const fs = require('fs');
+const URL = require('url').URL;
 
-const request = JSON.parse(process.argv[2]);
+const [, , ...args] = process.argv;
```

Comment:
I think this is one of the few places that could benefit from an inline comment. Now there are two ways of communication between Browsershot and Puppeteer via a Bash command, I'd list them here and explain in short what the code does. I believe this would be beneficial for future maintenance.

Reply:
Agreed